client/src/plugins/report-feedback/ClipboardCopySystemInfo.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import Metadata from '../../util/Metadata';
+
+const PLATFORM_MAPPING = {
+  'darwin': 'MacOS',
+  'win32': 'Windows',
+  'linux': 'Linux'
+};
+
+
+export class ClipboardCopySystemInfo {
+
+  constructor(props) {
+    const {
+      getGlobal,
+      activeTab
+    } = props;
+
+    this._backend = getGlobal('backend');
+    this._plugins = getGlobal('plugins');
+    this._config = getGlobal('config');
+    this._activeTab = activeTab;
+  }
+
+  async clipboardCopy(config) {
+    const systemInfo = await this._compileSystemInfoString(config);
+    this._backend.send('clipboard:write-text', { text: systemInfo });
```

Comment:
We have a copy function in the [Log](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/Log.js#L154) which is using browser API only. Is there any reason why we are using it differently here (via electron API)?

Reply:
Ok probably because we don't have selections here ...

Reply:
yep 

> Please check:
> 
> * color of the feedback icon is not aligned with other icons (e.g. deploy, run) –> should be #3B3B3B
> * change the color of the icon to white on active state (when bg is blue), see deploy + run for expected behavior
> * there is white space below the status bar; editor is scrollable –> please fix
> 
> The overlay is already awesome! Let's decide about an email address in the afternoon.

All resolved.

New build:
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2388-add-report-feedback/camunda-modeler-2388-add-report-feedback-linux-x64.tar.gz
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2388-add-report-feedback/camunda-modeler-2388-add-report-feedback-mac.dmg
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2388-add-report-feedback/camunda-modeler-2388-add-report-feedback-mac.zip
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2388-add-report-feedback/camunda-modeler-2388-add-report-feedback-win-ia32.zip
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2388-add-report-feedback/camunda-modeler-2388-add-report-feedback-win-x64.zip

Reply:
Hm, well we could have the text in some transient hidden area and select and copy from there.

But I think going via the designated electron API is not a bad thing is it?

Reply:
I would even argue using the electron API is the better approach 👍  I would use it instead of the browser API whenever possible. Everything else would look like a hack to me.

Just wanted to make sure we know that we have different ways of copying here 🙂

Reply:
Instead of doing `this._backend.send('clipboard:write-text', { text: systemInfo });` the least thing we could do is to move "copying to clipboard" behind an API we own (i.e. `remote`). This way it does not matter for a feature developer which way we do it.

So first important:

* Consolidate `copy` behind an API we own

Next:

* Use copy consistently (we use the API everywhere)

That would be best, if feasible of course.

Reply:
> Hm, well we could have the text in some transient hidden area and select and copy from there.

We don't need to do that. We can use asynchronous Clipboard API if we decide on the web way: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText

Reply:
This is how `remote/clipboard#writeText` would/could look like: https://github.com/camunda/camunda-modeler/pull/2400/commits/453ff786d3befc5649f1eb6a796a0b907130ad39